package.json

```diff
@@ -52,8 +53,8 @@
     "styled-components": ">= 5.1"
   },
   "dependencies": {
-    "@testing-library/testcafe": "^4.4.1",
-    "grommet-icons": "^4.8.0",
+    
```

Comment:
```suggestion
```
Good catch removing testcafe. Can we remove this extra line?
Everything else looks good